README.md

```diff
@@ -250,27 +250,17 @@ export NVM_DIR="$HOME/.nvm"
 
 By design and just like `scripts` defined in `package.json`, husky will run hook scripts as a single command. 
 
-```json
+```jsonc
 "pre-commit": "cmd && cmd"
-```
 
-That said, if you prefer to use an array, the recommended approach is to define them in `.huskyrc.js` or `husky.config.js`.
+// or
 
-```js
-const tasks = arr => arr.join(' && ')
-
-module.exports = {
-  'hooks': {
-    'pre-commit': tasks([
-      'cmd',
-      'cmd'
-    ])
-  }
-}
+"pre-commit": [
+  "cmd",
+  "cmd"
+]
 ```
 
-Tools like [npm-run-all](https://github.com/mysticatea/npm-run-all) can help too.
```

Comment:
I'm not entirely convinced the rest of this section can be removed. Especially the note on [npm-run-all](https://github.com/mysticatea/npm-run-all) can be useful as that tool allows you [to run commands in parallel](https://github.com/mysticatea/npm-run-all/blob/master/docs/run-p.md) as well :thinking:

Reply:
added back.